force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -285,12 +407,22 @@ public with sharing class FlowControlLoopsCollections {
      * Example: countMoney() should return 48.02
      * @return The total value in dollars, stopped at the point when it exceeds 40 dollars.
      */
-    public static Decimal countMoney() {
+      public static Decimal countMoney() {
         // The list of money in the wallet, represented in dollars and cents
         List<Decimal> moneyInWallet = new List<Decimal>{0.50, 10, 3.84, 24.60, 9.08, 50, 4.90}; //DO NOT CHANGE
-
-        return null; // Replace null with the variable you used to store the result
-    }
+        Decimal result = 0;
+        for (Integer i = 0; i < moneyInWallet.size(); i++){
+       
+        result = result + moneyInWallet[i];
+        System.debug('### moneyInWallet: ' + result); 
+                if (result > 40){   
```

Comment:
Try adding a variable for 40
https://en.wikipedia.org/wiki/Magic_number_(programming)